communication/marketing-team/README.md

```diff
@@ -1,57 +1,41 @@
-Future home for all things marketing team for contributor experience
-
-# Charter
-view our charter in this [marketing folder] for more about us and how to join
+The Contributor Marketing Team is part of the community-management subproject in
+[Contributor Experience] view our [charter] to learn more about us
 
 ## Purpose  
 to better inform the Kubernetes contributor community and highlight their work to
 the broader ecosystem
 
-# Core Marketing Team roles
+# Core Marketing Team
 
 | Role | Handbook | Notes |
 | --- | --- | --- |
-| Blogger(s) | TODO | |
-| Social Media Lead | TODO | for future contributor channels |
-| Social Media Team Members | TODO | |
-| Editor | TODO | unsure if this will be a position but could be good for making sure we are using the style guide and ensuring a consistent tone and brand; should be an approver in the communications folder |
-| Designer | TODO | help us with thumbnail graphics, digital assets, infographics, contributor site |
-| sig-docs blog community liaison | TODO link to sig-docs handbook | onlydole |
-| Internal Comms Lead | TODO | |  
-
-
-How to get in contact with us:  
-TODO: add slack team handle  
-tag us on your issues and PRs in kubernetes/community with `area/contributor-comms`
-
-# Focus Areas  
+| Editor | TODO | @parispittman (to bootstrap); assoc editor/shadow: @mbbroberg |
+| Internal Comms Lead | TODO | @jonasrosland; shadow: @simplytunde |
+| Social Media Lead | TODO | @chris-short; shadow - [Could be you!] (paris to help bootstrap) |
+| Bloggers | TODO | @onlydole (liaison to sig-docs-blog team) @celanthe, @vonguard, @boluisa, [Could be you!] |
+| Designer | TODO | [Could be you!] help us with thumbnail graphics, digital assets, infographics, contributor site |
+
```

Comment:
```suggestion
| Editor | TODO | Lead: @parispittman (to bootstrap); assoc editor/shadow: @mbbroberg |
| Internal Comms | TODO | Lead: @jonasrosland; shadow: @simplytunde |
| Social Media | TODO | Lead: @chris-short; shadow - [Could be you!] (paris to help bootstrap) |
| Bloggers | TODO | Lead: @onlydole (liaison to sig-docs-blog team); contributors: @celanthe, @vonguard, @boluisa, [Could be you!] |
| Designer | TODO | [Could be you!] help us with thumbnail graphics, digital assets, infographics, contributor site |

```

Reply:
I like adding the Leads here, but perhaps that could be a header in itself?

Reply:
Like so:
```
 | Role | Handbook | Lead | Shadow | Notes | 
```